internal/cmd/schema-registry/schema_utilities.go

```diff
@@ -73,6 +86,44 @@ func ReadSchemaRefs(cmd *cobra.Command) ([]srsdk.SchemaReference, error) {
 	return refs, nil
 }
 
+func ReadMetadata(flagName string, cmd *cobra.Command) (*srsdk.Metadata, error) {
```

Comment:
What's the reason for exporting this function? You only use it inside this package.
```suggestion
func readMetadata(flagName string, cmd *cobra.Command) (*srsdk.Metadata, error) {
```

Reply:
It's used in command_schema_create.go and command_subject_update.go